qt/widgets/common/src/WorkspacePresenter/WorkspaceTreeWidget.cpp

```diff
@@ -1148,6 +1154,29 @@ void WorkspaceTreeWidget::onClickDeleteWorkspaces() {
   m_presenter->notifyFromView(ViewNotifiable::Flag::DeleteWorkspaces);
 }
 
+/**
+ * Gets confirmation from user that they meant to press clear workspaces button
+ * @return True if yes is pressed, false if no is pressed
+ */
+bool WorkspaceTreeWidget::clearWorkspacesConfirmation() const {
+  return askUserYesNo(
+      "Clear Workspaces",
+      "Are you sure you want to clear all workspaces and hidden workspaces?");
```

Comment:
I'm not sure if all the users know what hidden workspaces are, this can confuse them.
Whoever is aware of those, will not be surprised if they are deleted when the others are.
```suggestion
      "This will delete all the workspaces, are you sure?");
```